Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4413,16 +4507,14 @@ RefPtr<StyleValue const> Parser::parse_position_area_value(TokenStream<Component
         return nullptr;
 
     tokens.discard_whitespace();
-    if (!tokens.has_next_token()) {
+    auto second_keyword_value = parse_keyword_value(tokens);
+    if (!second_keyword_value) {
```

Comment:
It's conceivably possible for a `<position-area>` to be followed by an unrelated keyword, in which case we shouldn't consume it, and should just parse the first keyword value. Could you check for that with a transaction here? Maybe it doesn't happen in practice, but this feels like a future footgun.

Reply:
If the second keyword is not a valid keyword all of the branches below will fail and the `return nullptr` at the bottom of the method will be hit. A transaction has already been started above which is committed by `create_keyword_list()`. 

Not sure if I'm missing something, but I don't think any change is needed here?

Reply:
I'm imagining some future property that could take a grammar like `<position-area> || wiggle`. We wouldn't want `<position-area>` to consume `wiggle`, just because it happens to be a keyword.

But yeah it's not based on a real situation I think.